src/main/java/com/icthh/xm/tmf/ms/customer/service/impl/CustomerServiceImpl.java

```diff
@@ -16,18 +11,21 @@
 import com.icthh.xm.tmf.ms.customer.repository.CustomerCharacteristicRepository;
 import com.icthh.xm.tmf.ms.customer.service.CustomerConfigurationService;
 import com.icthh.xm.tmf.ms.customer.service.CustomerService;
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.stream.Collectors;
-import javax.transaction.Transactional;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.data.jpa.domain.Specification;
 import org.springframework.stereotype.Service;
 import org.springframework.util.CollectionUtils;
 
+import javax.transaction.Transactional;
+import java.util.*;
```

Comment:
Wildcard imports should not be used according to our checkstyle

Reply:
Fixed